modules/db.js

```diff
@@ -205,6 +217,13 @@ module.exports = {
 		return await (client || pool).query("INSERT INTO Members (user_id, name, position, avatar_url, brackets, posts, show_brackets) VALUES ($1::VARCHAR(32), $2, (SELECT GREATEST(COUNT(position),MAX(position)+1) FROM Members WHERE user_id = $1::VARCHAR(32)), $3, $4, 0, false)", [userID,member.name,member.avatarURL || "https://i.imgur.com/ZpijZpg.png",member.brackets]);
 	},
 
+	addAuto: async (userID, member, client) => {
```

Comment:
These methods do not need `client` - that's typically only used for import / export.